src/ProjectTemplates/BlazorTemplates.Tests/playwrightSettings.debug.json

```diff
@@ -0,0 +1,7 @@
+{
+  "TimeoutInMilliseconds": 120000000,
```

Comment:
33 hours?

Reply:
Just a large number so that it doesn't get in the way when you are debugging.